examples/pub-sub-message-encryption/javascript/README.md

```diff
@@ -0,0 +1,39 @@
+# Sending and receiving messages with encryption
+
+This folder contains the code for encryption of messages (Typescript) - a demo of how you can encrypt and decrypt messages with [Ably](https://ably.com/docs/channels/options/encryption).
+
+## Getting started
+
+1. Clone the [Ably docs](https://github.com/ably/docs) repository where this example can be found:
+
+```sh
+git clone git@github.com:ably/docs.git
+```
+
+2. Change directory:
+
+```sh
+cd /examples/pub-sub-message-encryption/javascript/
+```
+
+3. Rename the environment file:
+
+```sh
+mv .env.example .env.local
+```
+
+4. In `.env.local` update the value of `VITE_PUBLIC_API_KEY` to be your Ably API key.
+
+5. Install dependencies:
+
+```sh
+yarn install
```

Comment:
Maybe it's better not to use `yarn`, so people won't have to additionally install it